ddtrace/tracer/util.go

```diff
@@ -63,3 +65,59 @@ func parseUint64(str string) (uint64, error) {
 	}
 	return strconv.ParseUint(str, 10, 64)
 }
+
+func isValidPropagatableTraceTag(k, v string) error {
```

Comment:
Normalising is generally done only in the agent. Let's not attempt at it here.

Reply:
It's not really a normalization. It's for checking if tags contain any invalid characters before we pass down the tags.
I can remove it. But, in that case, it's possible to pass a broken data into downstream apps.
As of today, we don't allow customers to set tag, so we don't expect any invalid tags. So, this check should always return `true` unless there is a bug.